### PR TITLE
KatjaHoffmann.TViewer version 2.3.0.5

### PR DESCRIPTION
**Pull request by katjah-dev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/83393)

### CODE REVIEW

**Comment by katjah-dev** [comment]
@microsoft-github-policy-service agree

**Comment by katjah-dev** [comment]
@microsoft-github-policy-service agree

File: manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: KatjaHoffmann.TViewer
+PackageVersion: 2.3.0.5
+PackageLocale: de-DE
+Publisher: Katja Hoffmann
+PackageName: TViewer
+License: proprietary
```
Comment (Trenly) [inline manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: KatjaHoffmann.TViewer
+PackageVersion: 2.3.0.5
+PackageLocale: de-DE
+Publisher: Katja Hoffmann
+PackageName: TViewer
+License: proprietary
+Copyright: Copyright © 2006-2021 Katja Hoffmann
+ShortDescription: TViewer Setup
```
Comment (Trenly) [inline manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml:11]: Please provide a better description of what TViewer does

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml (hunk 2 above)
Comment (katjah-dev) [reply manifests/k/KatjaHoffmann/TViewer/2.3.0.5/KatjaHoffmann.TViewer.locale.de-DE.yaml:11]: TViewer is a program to visualize the results of german equestrian competions based on the data of the Toris (it's the program, used to manage equestrian competitions program.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**